src/VscProcess.cpp

```diff
@@ -77,6 +77,30 @@ VscProcess::VscProcess() : myEStopState(0)
     }
   }
 
+  vsc_scm_target_set(vscInterface, 0);
+  vsc_scm_target_get(vscInterface);
+
+  ros::Duration(0.1).sleep();
+
+  vsc_setup_unlock(vscInterface);
+  vsc_get_setting(vscInterface, VSC_SETUP_KEY_RADIO_POWER_LEVEL);
+  vsc_get_setting(vscInterface, VSC_SETUP_KEY_SERIAL);
+  vsc_get_setting(vscInterface, VSC_SETUP_KEY_FIRMWARE);
+  
+  ros::Duration(0.1).sleep();
+
+  vsc_setup_unlock(vscInterface);
+  vsc_get_setting_int(vscInterface, VSC_SETUP_KEY_RADIO_POWER_LEVEL);
+  vsc_get_setting_int(vscInterface, VSC_SETUP_KEY_SERIAL);
+  vsc_get_setting_int(vscInterface, VSC_SETUP_KEY_FIRMWARE);
+  
+  ros::Duration(0.1).sleep();
+
+  vsc_setup_unlock(vscInterface);
+  vsc_get_setting_string(vscInterface, VSC_SETUP_KEY_RADIO_POWER_LEVEL);
+  vsc_get_setting_string(vscInterface, VSC_SETUP_KEY_SERIAL);
+  vsc_get_setting_string(vscInterface, VSC_SETUP_KEY_FIRMWARE);
+
```

Comment:
I understand those sleeps aren't very long. 
Wondering, Is there a way to wait for an `ACK-ACK` or `ACK-NACK` instead of sleeping?